libraries/cyclestreets-view/src/main/java/net/cyclestreets/util/Permissions.java

```diff
@@ -0,0 +1,16 @@
+package net.cyclestreets.util;
+
+import android.app.Activity;
+import android.content.pm.PackageManager;
+
+public class Permissions {
+  public static boolean verify(final Activity activity, final String permission) {
+    if (!hasPermssion(activity, permission))
+      activity.requestPermissions(new String[]{ permission }, 1);
+    return hasPermssion(activity, permission);
+  } // verify
+
+  private static boolean hasPermssion(final Activity activity, final String permission) {
```

Comment:
Can we please correct the missing `i` typo in this method name?

Reply:
I addressed this also in https://github.com/jezhiggins/cyclestreets-android/pull/48.